dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -97,6 +98,11 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
                 pushChannelReceived = co_await channelManager.CreatePushNotificationChannelForApplicationAsync();
 
+                PushNotificationTelemetry::ChannelRequestedByApi(
+                    S_OK,
+                    AppModel::Identity::IsPackagedProcess(),
```

Comment:
We can cache this value at the beginning of the function instead of doing two calls to this API.

Reply:
But won't the first use of the API disapear once unpackaged has been implemented?

Reply:
Thanks for pointing it out. Then the approach as is it's fine.